lib/realtime/repo.ex

```diff
@@ -52,41 +52,39 @@ defmodule Realtime.Repo do
   end
 
   @doc """
-  Converts a Postgrex.Result into a given struct
+  Deletes all records for a given query and converts them into a given struct
```

Comment:
the description doesn't seem to match the implementation, we're returning a number of deleted rows, not a struct.